shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-core/src/test/java/org/apache/shardingsphere/encrypt/metadata/EncryptTableMetaDataBuilderTest.java

```diff
@@ -212,16 +190,14 @@ public void assertLoadByExistedTablesH2() throws SQLException {
         EncryptTableMetaDataBuilder loader = (EncryptTableMetaDataBuilder) OrderedSPIRegistry.getRegisteredServices(RuleBasedTableMetaDataBuilder.class, rules).get(encryptRule);
```

Comment:
@tuichenchuxin This code appears in multiple test cases, can it be extracted into one method?